airbyte-integrations/connectors/source-leadfeeder/source_leadfeeder/manifest.yaml

```diff
@@ -0,0 +1,204 @@
+version: "0.29.0"
+
+definitions:
+  base_requester:
+    type: HttpRequester
+    url_base: https://api.leadfeeder.com
+    http_method: GET
+    request_headers:
+      Authorization: Token token={{ config["api_token"] }}
+  record_selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path:
+        - data
+  error_handler:
+    type: CompositeErrorHandler
+    error_handlers:
+      - type: DefaultErrorHandler
+        response_filters:
+          - type: HttpResponseFilter
+            action: IGNORE
+            http_codes:
+              - 402
+      - type: DefaultErrorHandler
+        max_retries: 2
+        response_filters:
+          - type: HttpResponseFilter
+            action: RETRY
+            http_codes:
+              - 429
+        backoff_strategies:
+          - type: ConstantBackoffStrategy
+            backoff_time_in_seconds: 60
+  partition_router:
+    type: SubstreamPartitionRouter
+    parent_stream_configs:
+      - type: ParentStreamConfig
+        stream:
+          $ref: "#/definitions/accounts_stream"
+        parent_key: id
+        partition_field: account_id
+  paginator:
+    type: DefaultPaginator
+    page_size_option:
+      type: RequestOption
+      field_name: page[size]
+      inject_into: request_parameter
+    page_token_option:
+      type: RequestOption
+      field_name: page[number]
+      inject_into: request_parameter
+    pagination_strategy:
+      type: PageIncrement
+      page_size: 100
+      start_from_page: 1
+      inject_on_first_request: true
+  incremental_sync:
+    type: DatetimeBasedCursor
+    start_datetime:
+      type: MinMaxDatetime
+      datetime: '{{ config["start_date"] }}'
+      datetime_format: "%Y-%m-%dT%H:%M:%SZ"
+    start_time_option:
+      type: RequestOption
+      field_name: start_date
+      inject_into: request_parameter
+    end_time_option:
+      type: RequestOption
+      field_name: end_date
+      inject_into: request_parameter
+    end_datetime:
+      type: MinMaxDatetime
+      datetime: "{{ now_utc().strftime('%Y-%m-%dT%H:%M:%SZ') }}"
+      datetime_format: "%Y-%m-%dT%H:%M:%SZ"
+    step: P1M
+    cursor_granularity: P1D
+
+  accounts_stream:
+    type: DeclarativeStream
+    name: accounts
+    primary_key:
+      - id
+    retriever:
+      type: SimpleRetriever
+      requester:
+        $ref: "#/definitions/base_requester"
+        path: /accounts
+        error_handler:
+          $ref: "#/definitions/error_handler"
+      record_selector:
+        $ref: "#/definitions/record_selector"
+    schema_loader:
+      type: JsonFileSchemaLoader
+      file_path: "./source_leadfeeder/schemas/accounts.json"
```

Comment:
Would be possible to integrate the schema inline or have it completly made by the builder?